src/modules/dashboard/components/Extensions/ExtensionCard.css

```diff
@@ -0,0 +1,30 @@
+.main {
+  height: 220px;
+  width: 250px;
+}
+
+.card {
+  display: block;
+  padding: 15px;
+  height: 100%;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.headerIcon {
```

Comment:
Can you please remove the `stroke` on the underlying `svg` element

![Screenshot from 2020-11-17 14-17-12](https://user-images.githubusercontent.com/1193222/99389371-a1a9a600-28df-11eb-9608-39ac855808e3.png)

This might not be needed after you update the the icon with the new version